tests/core/test_actions.py

```diff
@@ -115,11 +116,12 @@ def test_domain_action_instantiation():
     assert instantiated_actions[6].name() == ACTION_DEFAULT_ASK_AFFIRMATION_NAME
     assert instantiated_actions[7].name() == ACTION_DEFAULT_ASK_REPHRASE_NAME
     assert instantiated_actions[8].name() == ACTION_TWO_STAGE_FALLBACK_NAME
-    assert instantiated_actions[9].name() == ACTION_BACK_NAME
-    assert instantiated_actions[10].name() == RULE_SNIPPET_ACTION_NAME
-    assert instantiated_actions[11].name() == "my_module.ActionTest"
-    assert instantiated_actions[12].name() == "utter_test"
-    assert instantiated_actions[13].name() == "utter_chitchat"
+    assert instantiated_actions[9].name() == ACTION_UNLIKELY_INTENT_NAME
```

Comment:
How did you know this action would be at index 9? 🤔

Reply:
`ACTION_UNLIKELY_INTENT_NAME` is a default action which means it's added to `DEFAULT_ACTION_NAMES` explicitly. We know its position (the index is 9) which means this test simply checks if all the default actions were added to the list of instantiated actions.

See `DEFAULT_ACTION_NAMES` in `rasa/shared/core/constants.py` and `_combine_user_with_default_actions` in `rasa/shared/core/domain.py` for more details.